.github/actions/github-release/main.js

```diff
@@ -91,13 +91,25 @@ async function runOnce() {
   } catch (e) {
     console.log("ERROR: ", JSON.stringify(e, null, 2));
     core.info(`creating a release`);
+
+    const releaseNotes = fs.readFileSync('RELEASES.md').toString();
+    let notes = null;
+    const opts = {
+      owner,
+      repo,
+      tag_name: name,
+      prerelease: name === 'dev',
+    };
+    if (name !== 'dev') {
+      for (let x of releaseNotes.split(/^-+$/m)) {
```

Comment:
Do we need to be more defensive? What about lists that start a line with `-` or a `--` that somehow gets formatted to the beginning of a line?

```suggestion
      for (let x of releaseNotes.split(/^---+$/m)) {
```